sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -73,349 +73,3 @@ private[sql] case class ExecutedCommand(cmd: RunnableCommand) extends SparkPlan
 
```

Comment:
Now that you have the `command` package, would it make sense to move `RunnableCommand` there? Right now it's confusing how there's a `o.a.s.sql.execution.commands.scala` and a `o.a.s.sql.execution.command.commands.scala`. We should only have the latter and move everything here there.